signer/xpi/x509.go

```diff
@@ -133,14 +133,32 @@ func (s *XPISigner) generateIssuerEEKeyPair() (eeKey crypto.PrivateKey, eePublic
 			return
 		}
 
+		newKey, ok := eeKey.(*rsa.PrivateKey)
```

Comment:
we could try to dedup these case statements with very similar logic for rsa and ecdsa key types. e.g. pull out fns for `getIssuerRSAKeySize` and `getIssuerECDSACurve`

Reply:
fixed in 8c283b1c5ad8cb84f8985d71df9cf46f0212d4d0